appUITests/Screens/BaseScreen.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  UIConstants.swift
+//  GoldenSampleUITests
+//
+//  Created by Backbase on 15/12/2023.
+//
+
+import Foundation
+import XCTest
+
+class BaseScreen {
+    
+    //MARK: ELEMENTS
+    var app = XCUIApplication()
```

Comment:
@maryna-bb i set the app init once at this class, it will only exposed to subclasses. also subclasses should refer to that constant app too without extra init. let me know if this is not sufficient.

@ashchupliak retail MB is using 
```
var app: XCUIApplication

init(_ app: XCUIApplication) {
        self.app = app
    }
```